html/browsers/windows/auxiliary-browsing-contexts/opener-closed.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<meta charset="utf-8">
+<html>
+  <head>
+    <title>Auxiliary Browing Contexts: window.opener when Opener Removed/Closed</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+  <body>
+    <div id="log"></div>
+    <script>
+    setup (() => {
+      localStorage.clear();
```

Comment:
I think this could affect other tests if one browser instance is running multiple tests in parallel (or the same test in multiple tabs). I'm not sure if that is something that happens today but at least it was a problem for Opera's testing infrastructure a few years ago.

I would recommend generating a new unique (enough) ID each time a test is run, and use that in they key for `localStorage`. For example:
```
window.testId = `${location.pathname}-${Math.random()}-${Date.now()}`;
delete localStorage[testId + '-closeAll'];
```
You would then need to pass the `testId` to the support files that need it, e.g. by including it in the query string.